aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,26 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## June 4, 2022 - July 8, 2022
+
+### Stay Current with the Latest Changes
+
+- **Tabs**
+  - We've enhanced responsive behavior and updated the guidance.
+- **Menu**
+  - We've added item grouping and added guidance for when to use.
+
+### Roadmap Updates
+
+- **Figma Component Refactoring**
+  - We're just about ready to announce the first version new input components, hopefully next week. Thanks for all of the help we received.
```

Comment:
personally I would take out the next week just in case we don't make promises and then an unforeseen circumstance happens

Reply:
done

Reply:
thank you! 

think we might have to update yarn..